packages/camera/camera_web/example/lib/main.dart

```diff
@@ -2,12 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: public_member_api_docs
```

Comment:
This shouldn't be added; we want documentation in examples.

I know there are some files that have it, but that's a legacy issue that we shouldn't expand.